en/modules/admin/pages/initiatives.adoc

```diff
@@ -77,23 +77,23 @@ image:admin_initiative_type.png[New initiative type form]
 
 |Only allow global scope initiatives creation
 |Optional
-|Tick this flag if you enabled "Child scope signature" and configured the global scope as your parent scope. By enabling this, initiative type selection will be skipped in the initiative creation wizard.
+|Tick this flag if you enabled "Child scope signature" and configured the global scope as your parent scope. By enabling this, scope selection will be skipped in the initiative creation wizard.
 
 |Enable promoting committee
 |Optional
-|Whether it will only allow individual initiatives (made by only one person) or it will always have to have a collective (multiple promoters, being multiple individuals or a user group).
+|Whether it will only allow individual initiatives (made by only one person) or allow collective initiatives (multiple promoters, being multiple individuals or a user group).
 
 |Minimum of committee members
 |Optional
-|Depends on whether the "Enable promoting committee" option is checked
+|When "Enable promoting committee" option is checked, defines the minimum number of promoter's committee member to send the initiative to technical validation
 
 |Collect participant personal data on signature
 |Optional
-|When a participant is the signature process, it'll ask for some personal data. This will be used with the PDF export action after the recollection period has ended.
+|Adds a step to the signature process. When a participant clicks on the signature button, she'll be redirected to a form asking for some personal data. This will be used with the PDF export action after the recollection period has ended.
 
 |Legal information about the collection of personal data
 |Depends on whether the option "Collect participant personal data on the signature" is checked
-|Text that it'll show the participant during the signature process, for privacy notice and terms of use of this personal data.
+|Inputed text that will be displayed to the participant during the signature process, for privacy notice and terms of use of this personal data.
```

Comment:
I don't like "inputed text", as all the HTML forms are inputs texts. It's too technical sounding. Maybe something like:
```suggestion
|Privacy notice and terms of use for the participant personal data during the signature process.
```
?